backend/.env

```diff
@@ -10,8 +10,8 @@ OTHER_REACT_APP_KYC_PROPERTIES_TO_COMPARE= //birthdate,email,phone_number,addres
 ESIGNET_USERINFO_URL = 'https://esignet.collab.mosip.net/v1/esignet/oidc/userinfo'
 PORT=8080
 HOST="localhost"
-USER="root"
-PASSWORD=""
+USER="user"
+PASSWORD="pass123"
```

Comment:
need to move away from plaintext passwords in config and find a way to inject during deploy time / apply custom config or use keys.